tests/test_partisn.py

```diff
@@ -397,10 +397,10 @@ def write_partisn_input_1D():
     ngroup = 5
 
     # expected output file
-    import platform
-    if platform.python_version().split('.')[0] == '2':
+    import sys
+    if sys.version_info[0] == 2 :
         file_expected = os.path.join(THIS_DIR, 'files_test_partisn', 'partisn_1D_expected-2.inp')
-    elif platform.python_version().split('.')[0] == '3':
+    elif sys.version_info[0] == 3 :
```

Comment:
for all of these tests that check two different versions of the file depending on the python version, we should see if now the two files are actually identical. I suspect they might be since I added `sorted()` to anywhere where dictionaries are written out. I just didn't have a reasonable way to test if that was the case before.